src/menu.c

```diff
@@ -20,31 +20,41 @@
 #include "settings.h"
 #include "utils.h"
 
+#define ERROR_BUF_LEN 120
+
 static bool is_initialized = false;
 static regex_t cregex;
+static char ERROR_BUF[ERROR_BUF_LEN];
```

Comment:
I'd suggest using a local variable for this and without the `#define`.

```
char error_buf[120];
```

Reply:
Done.